pepper-apis/src/main/java/org/broadinstitute/ddp/model/activity/instance/PhysicianInstitutionComponent.java

```diff
@@ -20,56 +25,56 @@
 
     // fields are marked transient so that gson does not deserialize them.  instead,
     // they are added to the parameters list, which is serialized.
-    private transient boolean allowMultiple;
-
-    private transient String addButtonText;
+    private transient InstitutionPhysicianComponentDto instDto;
+    private transient boolean shouldHideNumber;
 
+    private transient String buttonText;
     private transient String titleText;
-
     private transient String subtitleText;
 
-    private transient InstitutionType institutionType;
-
-    private transient boolean showFields;
-
-    private transient boolean required;
-
-    protected PhysicianInstitutionComponent(ComponentType physicianOrInstitution,
-                                            boolean allowMultiple,
-                                            String addButtonText,
-                                            String titleText,
-                                            String subtitleText,
-                                            InstitutionType institutionType,
-                                            boolean showFields,
-                                            boolean required,
-                                            boolean hideNumber) {
+    protected PhysicianInstitutionComponent(
+            ComponentType physicianOrInstitution,
+            InstitutionPhysicianComponentDto instDto,
+            boolean shouldHideNumber
+    ) {
```

Comment:
Instead of passing all these parameters, we are now passing `InstitutionPhysicianComponentDto` that has all we need.